addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -74,6 +83,16 @@ var FormRenderer = BasicRenderer.extend({
                 return;
             }
         }
+        if (this.quickEditInfo) {
+            const { fieldName, extraInfo } = this.quickEditInfo;
+            this.quickEditInfo = null;
+            const fieldWidget = this.allFieldWidgets[this.state.id]
+                .find(field => field[symbol] === fieldName);
```

Comment:
as far as i understand, `field[symbol]` is the `node.id`. How can this match with a `fieldName`?

Reply:
`node.id` is the fieldName for the first occurrence of a field in the view, by for other occurrences it is the fieldName with a unique suffix...